etc/ci.sh

```diff
@@ -37,18 +31,62 @@ else
         make
         cd ../..
 
-        sh bin/gap.sh --cover coverage tst/${TEST_SUITE}.g
+        sh bin/gap.sh -q tst/extractmanuals.g
 
-        # generate coverage report
-        sh bin/gap.sh -q <<GAPInput
-            if LoadPackage("profiling") <> true then
-                Print("ERROR: could not load profiling package");
-                FORCE_QUIT_GAP(1);
-            fi;
-            OutputJsonCoverage("coverage", "coverage.json");
+        for ch in tst/testmanuals/*.tst
+        do
+            COVNAME="coverage.`basename $ch .tst`"
+            sh bin/gap.sh -q --cover $COVNAME <<GAPInput
```

Comment:
Even better: You could move this call *after* the `esac`, and loop over all `coverage*` files in GAP. I.e. it would be the same code for the `testmanuals` code as for all the other `testFOO` targets.